src/_locales/en/messages.json

```diff
@@ -243,6 +255,10 @@
     "message": "QR code size",
     "description": "The heading/grouping title shown for the radio buttons for the size setting of the QR code."
   },
+  "optionMisc": {
```

Comment:
Well… for consistency you should then also name this "title…`, should not you?

Reply:
> I used title to refer to the h1 text, this is one of the subtitles, so i used the current naming convention omstead

So let's keep the discussion here in this line.
Okay, fair enough. However, then maybe name it `subtitle` or so?

I mean we always have `description`, where we can explain it all in detail...